plugins/ltac/extratactics.mlg

```diff
@@ -146,6 +146,23 @@ let discrHyp id =
 let injection_main with_evars c =
  elimOnConstrWithHoles (injClause None None) with_evars c
 
+let warn_deprecated_injection_intropattern_syntax = fun ?loc () -> ()
+(* Keeping code in case the general view evolves (e.g. when "on H do pat"
+   is introduced) *)
+(*
+  CWarnings.create ~name:"injection-arguments-flat-intropattern-syntax" ~category:"deprecated"
+    (fun () ->
+       strbrk "Syntax " ++ str "\"injection ... as pat1 ... patn\"" ++
+       strbrk " is deprecated. Use " ++ str "\" injection ... as [= pat1 ... patn ]\".")
+*)
```

Comment:
Please don't do this: the code is not complex enough to be useful to keep, and commented code is bound to rot. If you want to keep a comment noting that it was discussed whether the simpler intro-pattern should be deprecated, just keep it as a textual comment.